LiveSched/src/main/java/dev/coms4156/project/livesched/Schedule.java

```diff
@@ -0,0 +1,73 @@
+package dev.coms4156.project.livesched;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Coordinates scheduling given a Task and a List of ResourceTypes.
+ * This class stores the available ResourceTypes and a Task
+ * and modifies their values upon scheduling
+ */
+public class Schedule {
+  private final String scheduleId;
+  private Task task;
+  private List<ResourceType> resourceTypes;
+  private List<ResourceType> candidates;
+
+  /**
+   * Constructs a new Task object with the given parameters.
+   *
+   * @param scheduleId        the unique ID of the schedule
+   * @param task              task to provide resource to
+   * @param resourceTypes     list of available resource types
+   * @param radius            maxinmum distance from task to resource
```

Comment:
I understand why we are using the name 'radius' but maybe we could use a different variable name like maxDistance to make it more clear?